src/Uno.Sdk/Sdk/Sdk.targets

```diff
@@ -29,6 +29,23 @@ Copyright (C) Uno Platform Inc. All rights reserved.
 		<Using Remove="@(Using->HasMetadata('Platform'))" />
 	</ItemGroup>
 
+	<ItemGroup Condition=" !$(IsWinAppSdk) ">
```

Comment:
In general .NET globbings do have a way to exclude them, like "DisableDefaultIncludes" or something similar. I don't recall if there's individual one per item group.

Reply:
ahh yes... good call there...